data/fields/marker.json

```
@@ -1,5 +1,33 @@
 {
     "key": "marker",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+            "aerial": {
+                "title": "Aerial",
+                "description": "Designed to be seen by aerial inspection."
```

This is a simpler way to say the same thing:

```suggestion
                "description": "Designed to be visible from the air."
```